lona/view.py

```diff
@@ -37,9 +37,8 @@ def _add_view_to_objects(cls, view_object):
     def _remove_view_from_objects(cls, view_object):
         objects_key = cls._get_objects_key(view_object, is_class=False)
 
-        if(objects_key in cls._objects and
-           view_object in cls._objects[objects_key]):
-
+        if (objects_key in cls._objects and
+                view_object in cls._objects[objects_key]):
```

Comment:
This is also only a style change

Reply:
Removed this change. It was done by my IDE, not intentionally.